scripts/release_notes.rb

```diff
@@ -78,7 +83,11 @@ def gather_issues(offset = 0)
   puts "#### #{category}"
 
   issues.each do |issue|
-    puts "* #{issue['subject'].gsub('`','\\\`').gsub('<','&lt;').gsub('>','&gt;').gsub('*','\\\*')} \\- [##{issue['id']}](#{URL}/issues/#{issue['id']})"
+    if @output == "markdown"
+      puts "* #{issue['subject'].gsub('`','\\\`').gsub('<','&lt;').gsub('>','&gt;').gsub('*','\\\*')} \\- [##{issue['id']}](#{URL}/issues/#{issue['id']})"
+    else
+      puts "* #{issue['subject'].gsub('`','\\\`').gsub('<','&lt;').gsub('>','&gt;').gsub('*','\\\*')} \\- #{URL}/issues/#{issue['id']}[##{issue['id']}]"
```

Comment:
I think we dont need escape char before `-` as I see `\-` is getting rendered on: https://asciidoclive.com/edit/scratch/1 with,
~~~
* double Puppet deprecation notice on host creation page \- https://projects.theforeman.org/issues/32493[#32493]
~~~
```suggestion
      puts "* #{issue['subject'].gsub('`','\\\`').gsub('<','&lt;').gsub('>','&gt;').gsub('*','\\\*')} - #{URL}/issues/#{issue['id']}[##{issue['id']}]"
```

Reply:
Thanks I was rushing out quick and it was completely untested.